packages/sso-express/src/controllers.ts

```diff
@@ -124,6 +134,4 @@ export const authCallbackController =
       console.error(err);
       res.redirect(options.oidcConfig.baseUrl);
     }
-
-    next();
```

Comment:
Makes sense to not call `next()` as the response is already sent. 
However, this is a feature that is currently used and needs to be replaced by an equivalent, e.g. an `onPostLogin(claims)` callback.